autoload.php

```diff
@@ -7,9 +7,17 @@
         STDERR,
         "\n==== Redirecting to Composer-installed version in vendor/codeception. You can skip this using --no-redirect ====\n"
     );
-    require $autoloadFile;
-    //require package/bin instead of codecept to avoid printing hashbang line
-    require './vendor/codeception/codeception/package/bin';
+
+    if (file_exists('./vendor/codeception/codeception/app.php')) {
```

Comment:
This seems redundant since even in 4+ we have a `package/bin` file (which is also changed in this PR).

Reply:
`package/bin` must be kept for backwards compatibility with codeception v3 phar files.
This condition is good for forward compatibility -  when `package/bin` is deleted from Codeception v5 or v6 and Codeception 4.0 phar file tries to load it.